Namco/pacman.py

```diff
@@ -29,14 +29,28 @@ def step(self):
     def render(self):
         pass
 
+    # Selecting automatic action if no user input (direction) given
+    def select_action(self, prev_action):
+        user_input, timedOut = timedInput("Choose your next action:\n")
+        if timedOut:
+            action = prev_action
+        else:
+            action = user_input
```

Comment:
Esetleg itt megnézhetnénk, hogy a user input-ja w,a,s vagy d-e és ha nem akkor kiírhatjuk, hogy "Invalid input, use W, A, S or D." És ha nem ezek valamelyikét írta be, akkor ne update-lejük az actiont.